src/main/java/fr/moribus/imageonmap/commands/IoMCommand.java

```diff
@@ -43,11 +43,27 @@
 import fr.zcraft.quartzlib.components.i18n.I;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.UUID;
+import java.util.function.Consumer;
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
 import org.bukkit.entity.Player;
 
 
 public abstract class IoMCommand extends Command {
 
+
+    protected void consumePlayerCommand(String arg, Consumer<UUID> consumer) {
```

Comment:
The method Bukkit.getOfflinePlayer(String) is deprecated, an equivalent is:  `@Nullable
public OfflinePlayer getOfflinePlayer(@Nullable final String name) {
  return ArrayList(Bukkit.getOfflinePlayers()).stream().filter(p -> p.getName().equals(name)).findFirst().orElse(null);
}` This is not usefull right know and may be added in quartzlib.(which will result in a futur PR to change this)